jit-application/pom.xml

```diff
@@ -70,6 +70,48 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
please manage all versions in the parent pom, that way if we want to move some of it to super-pom it will be easier (and I think we do want to move some of these to super-pom)

Reply:
Also note that at least camel has a bom that can be used instead of specifying every dependency manually